### ISSUE TITLE
Error at /mt/marian/src/kernels/dropout.cu:22

### ISSUE THREAD

**Issue by kpu** [amunmt/marian#46]
From current master, seems to break no matter what I do (on Azure):

/mt/marian/build/marian -t data/corpus.bpe.ro data/corpus.bpe.en -d 0
[2017-03-27 16:43:16] [config] after-batches: 0
[2017-03-27 16:43:16] [config] after-epochs: 0
[2017-03-27 16:43:16] [config] clip-norm: 1
[2017-03-27 16:43:16] [config] devices:
[2017-03-27 16:43:16] [config]   - 0
[2017-03-27 16:43:16] [config] dim-emb: 512
[2017-03-27 16:43:16] [config] dim-rnn: 1024
[2017-03-27 16:43:16] [config] dim-vocabs:
[2017-03-27 16:43:16] [config]   - 50000
[2017-03-27 16:43:16] [config]   - 50000
[2017-03-27 16:43:16] [config] disp-freq: 1000
[2017-03-27 16:43:16] [config] dropout-rnn: 0
[2017-03-27 16:43:16] [config] dropout-src: 0
[2017-03-27 16:43:16] [config] dropout-trg: 0
[2017-03-27 16:43:16] [config] early-stopping: 10
[2017-03-27 16:43:16] [config] layer-normalization: false
[2017-03-27 16:43:16] [config] layers-dec: 1
[2017-03-27 16:43:16] [config] layers-enc: 1
[2017-03-27 16:43:16] [config] learn-rate: 0.0001
[2017-03-27 16:43:16] [config] max-length: 50
[2017-03-27 16:43:16] [config] maxi-batch: 100
[2017-03-27 16:43:16] [config] mini-batch: 64
[2017-03-27 16:43:16] [config] model: model.npz
[2017-03-27 16:43:16] [config] no-reload: false
[2017-03-27 16:43:16] [config] no-shuffle: false
[2017-03-27 16:43:16] [config] optimizer: adam
[2017-03-27 16:43:16] [config] overwrite: false
[2017-03-27 16:43:16] [config] relative-paths: false
[2017-03-27 16:43:16] [config] save-freq: 10000
[2017-03-27 16:43:16] [config] seed: 1234
[2017-03-27 16:43:16] [config] skip: false
[2017-03-27 16:43:16] [config] train-sets:
[2017-03-27 16:43:16] [config]   - data/corpus.bpe.ro
[2017-03-27 16:43:16] [config]   - data/corpus.bpe.en
[2017-03-27 16:43:16] [config] type: dl4mt
[2017-03-27 16:43:16] [config] valid-freq: 10000
[2017-03-27 16:43:16] [config] valid-metrics:
[2017-03-27 16:43:16] [config]   - cross-entropy
[2017-03-27 16:43:16] [config] workspace: 2048
[2017-03-27 16:43:16] [data] Loading vocabulary from data/corpus.bpe.ro.json (max: 50000)
[2017-03-27 16:43:16] [data] Loading vocabulary from data/corpus.bpe.en.json (max: 50000)
Error at /mt/marian/src/kernels/dropout.cu:22

**Comment by kpu** [comment]
Oops, this appears to be the error for driver/userspace mismatch.  Perhaps a better error message is in order but it's not Marian's fault.

**Comment by emjotde** [comment]
Ah, I was just writing that it is unlikely it's our fault, standard curand function. Maybe you downgrade to 367.48 as I suggested in issue #45 ?
